astropath/path.py

```diff
@@ -94,14 +94,15 @@ def init_cand_prior(self, P_O_method:str, P_U=0.):
         # Vet
         assert priors.vet_cand_prior(self.cand_prior, self.candidates), 'Bad candidate prior input'
 
-    def init_theta_prior(self, PDF:str, max:float):
+    def init_theta_prior(self, PDF:str, max:float, scale:float):
         """Ingest the theta (offset) prior
 
         Args:
             PDF (str): Method
             max (float): Maximum offset allowed
+            scale (float): Scaling of phi [only for exp method, so far]
```

Comment:
Add example. e.g., "scale = 2.0 for exp(-2*theta/phi)"

Reply:
good suggestion.  added!